### PR TITLE
Creating Joomla template

### PR DESCRIPTION
**Pull request by chrisft25**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v2/apps/joomla.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "captainVersion": "2",
+    "documentation": "Taken from https://hub.docker.com/_/joomla?tab=description. Based on Wordpress - No Database template.",
+    "displayName": "Joomla - No Database",
+    "description": "This will create a Joomla only. You will need to create and configure the database information manually. Intended for advanced users.",
+    "dockerCompose": {
+        "version": "3.3",
+        "services": {
+            "$$cap_appname-joomla": {
+                "image": "joomla:latest",
```
Comment (githubsaturn) [inline public/v2/apps/joomla.json:10]: image

Please avoid using latest tag, and use `$$cap_joomla_version` that you created below. This is part of the checklist that you checked.

Comment (chrisft25) [reply public/v2/apps/joomla.json]: You're right! I forgot to add the variable to the image. I have done it already!

Comment (githubsaturn) [reply public/v2/apps/joomla.json]: @chrisft25 - thanks! But the variable's default value is still set to `latest` - we have to change this to a fixed value.

Comment (chrisft25) [reply public/v2/apps/joomla.json]: It's done!

**Comment by githubsaturn** [comment]
Great, thanks!

**Pull request merged**